components/button-cards/solution-card/solution-card.config.yml

```diff
@@ -0,0 +1,12 @@
+context:
+  general:
+    cardDescription: 'Solution Cards'
```

Comment:
I think we should move the markup text into the yml file, or delete all these properties, since they aren't needed